packages/orbit-components/src/Loading/Loading.stories.tsx

```diff
@@ -1,123 +1,152 @@
+  argTypes: {
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
+    text: {
+      control: {
+        type: "text",
+      },
+    },
   },
 };
 
-export const CardLoading = ({ title, description, loading, loadingText }) => {
-  return (
-    <Card loading={loading} title={title} description={description}>
-      <Loading loading={loading} type="boxLoader" text={loadingText}>
-        <CardSection>
-          <Illustration name="EnjoyApp" size="medium" />
-        </CardSection>
-      </Loading>
-    </Card>
-  );
-};
+export default meta;
 
-CardLoading.story = {
-  name: "Card loading",
+type Story = StoryObj<typeof Loading>;
 
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+export const Default: Story = {
+  args: {
+    customSize: undefined,
+    loading: undefined,
+    text: undefined,
+    type: undefined,
   },
-};
-
-CardLoading.args = {
-  title: "Card with title",
-  description: "",
-  loading: true,
-  loadingText: "Please wait, Card content is loading...",
-};
 
-export const InlineLoader = ({ loadingText }) => {
-  return <Loading type={TYPE_OPTIONS.INLINE_LOADER} text={loadingText} />;
+  parameters: {
+    controls: {
+      disable: true,
+    },
+  },
 };
 
-InlineLoader.story = {
-  name: "Inline loader",
+export const DefaultWithChildren: Story = {
+  render: args => (
+    <Loading {...args}>
+      <Text>Loading prop is false.</Text>
+    </Loading>
+  ),
 
   parameters: {
-    info: "This configuration of this component is without any height, width or paddings. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["children", "customSize", "type"],
+    },
   },
-};
 
-InlineLoader.args = {
-  loadingText: "Please wait, content of the page is loading...",
+  args: {
+    customSize: undefined,
+    type: undefined,
+  },
 };
 
-export const CustomLoader = ({ customSize }) => {
-  return <Loading customSize={customSize} />;
+type CardLoadingStoryProps = React.ComponentProps<typeof Loading> & {
+  cardTitle: string;
+  cardDescription: string;
 };
 
-CustomLoader.args = {
-  customSize: 50,
-};
+export const CardLoading: StoryObj<CardLoadingStoryProps> = {
+  render: ({ loading, cardTitle, cardDescription, ...args }) => {
+    return (
+      <Card loading={loading} title={cardTitle} description={cardDescription}>
+        <Loading {...args} type="boxLoader">
+          <CardSection>
+            <Illustration name="EnjoyApp" size="medium" />
+          </CardSection>
+        </Loading>
+      </Card>
+    );
+  },
 
-export const Playground = ({ type, loadingText, loading, dataTest }) => {
-  return <Loading loading={loading} type={type} text={loadingText} dataTest={dataTest} />;
-};
+  args: {
+    cardTitle: "Card title",
+    cardDescription: "Card description",
+    text: undefined,
+    customSize: undefined,
+  },
```

Comment:
Are all these args necessary? Do we need to have Card props in controls? I would just keep here the control for `loading` and hardcode the rest, WDYT?